### PR TITLE
RM-8913 ColumnTitleDisplayValue throws Exception for dynamic columns

### PR DESCRIPTION
**Pull request by martin-herwich**
https://re-motion.atlassian.net/browse/RM-8913

### CODE REVIEW

File: Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocSimpleColumnDefinitionTest.cs [hunk 1]
```diff
@@ -66,5 +69,46 @@ public void GetValidationFailureMatcher_ReturnsInstanceOfPropertyPathValidationF
           ((BusinessObjectPropertyPathValidationFailureMatcher)validationFailureMatcher).PropertyPath,
           Is.EqualTo(column.GetPropertyPath()));
     }
+
+    [Test]
+    public void ColumnTitleDisplayValue_EmptyColumnTitle_StaticPropertyPath_ReturnsDisplayNameOfLastProperty ()
```
Comment (Frazerus) [inline Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocSimpleColumnDefinitionTest.cs:74]: Do we also want a test for this case with a last property = null?

**Review by Frazerus — APPROVED** [review]

File: Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocSimpleColumnDefinitionTest.cs (hunk 1 above)
Comment (martin-herwich) [reply Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocSimpleColumnDefinitionTest.cs:85]: Yes, good idea. I added that.

**Review by MichaelKetting — APPROVED** [review]
